### PR TITLE
Sero's changes cleaned up

### PR DESCRIPTION
**Pull request by coolo**

### CODE REVIEW

**Comment by nilxam** [comment]
the definition of holiday has changed? :)

File: tests/qa_automation/qa_run.pm [hunk 1]
```diff
@@ -0,0 +1,100 @@
+package qa_run;
```
Comment (aaannz) [inline tests/qa_automation/qa_run.pm]: IMO this file should be in `lib/` dir. Or we'll need to add tests/qa_automation to `@INC` which I think is ugly.

**Comment by coolo** [comment]
holiday? you mean weekend? I had a week with 6 days of talking and no coding - I'm DONE WITH IT! :)

File: tests/qa_automation/qa_run.pm (hunk 1 above)
Comment (coolo) [inline tests/qa_automation/qa_run.pm]: I think moving qa_automation specific stuff to lib is ugly too - I have the same in firefox.pm and sle11_firefox.pm in x11/.

I don't want firefox.pm in lib/ either

Comment (coolo) [inline tests/qa_automation/qa_run.pm]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/4db8be211d1fdf0613fc570c8753dfd6c8f0e485see

Comment (aaannz) [inline tests/qa_automation/qa_run.pm]: Well yeah, that fixes it, but how about using i. e. `lib/automation` and `lib/x11`. And then in tests call `use base "automation/qa_run"`?

Comment (coolo) [inline tests/qa_automation/qa_run.pm]: but then firefox.pm is no longer a valid test on its own

Comment (aaannz) [inline tests/qa_automation/qa_run.pm]: Ah I see. Needed to check the code to see the usage. :)

**Comment by coolo** [comment]
I think it found a bug:
https://openqa.suse.de/tests/157393/modules/openssl/steps/11

**Pull request merged**